src/contexts/CollaborationProvider.tsx

```diff
@@ -65,6 +68,30 @@ export const CollaborationProviderContext = ({ documentId, document, children }:
   const [connected, setConnected] = useState<boolean>(false)
   const [provider, setProvider] = useState<HocuspocusProvider>()
 
+  // Developer tool to display current document source in a dialog
+  const { showModal, hideModal } = useModal()
+  useKeydownGlobal((evt) => {
+    if (evt.key === 'u' && evt.shiftKey && evt.metaKey) {
```

Comment:
I need this for it to work, don't know if it's OS specific
```suggestion
    if (evt.key === 'U' && evt.shiftKey && evt.metaKey) {
```

Reply:
and just a note, metaKey = windows key for me.

Reply:
I can't seem to make it work at all.